test/ssc_test/cmod_pvsamv1_test.cpp

```diff
@@ -666,7 +670,7 @@ TEST_F(CMPvsamv1PowerIntegration_cmod_pvsamv1, NoFinancialModelSystemDesign)
                                 192329, 187710, 185732, 197565, 201021, 192329, 187710
     };
 
-    for (size_t i = 0; i != annual_energy_expected.size(); i++)
+    for (size_t i = 57; i < annual_energy_expected.size(); i++)
```

Comment:
I'm looking at coveralls and trying to find the coverage decrease, and given that the mismatch losses are no longer covered I think this a likely candidate:

![image](https://github.com/user-attachments/assets/f1cfd6e1-60f8-4aa7-b2e9-e42c6cc10bc0)

Mismatch losses appear to be case i = 13. Can you:

(a) Edit the arrays above to only include the cases we want to keep? 
(b) Ensure mismatch losses are included in those cases?

Reply:
Great catch - running i=13 gives the same coverage as running all 64 configurations

base run (no changes from the i=57 to annual_energy_expected.size()
![image](https://github.com/user-attachments/assets/0dcb9811-9d77-4725-9dc9-017171e47f19)

i=13 results - coverage increased and time reduced
![image](https://github.com/user-attachments/assets/fc30f174-5a7b-4065-b5fb-4c8842bd1f14)

enabling all tests does not increase coverage and a 2.5 times increase in elapsed time
![image](https://github.com/user-attachments/assets/2e23828f-f8f2-4a19-a2b3-116d148125c4)

Updated in latest commit and commented code to be removed after one additional test